### PR TITLE
misc fixes

### PR DESCRIPTION
**Pull request by rtisma**
test

### CODE REVIEW

File: song-client/src/main/java/org/icgc/dcc/song/client/errors/ServerErrors.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+import lombok.Getter;
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+
+@Getter
+@RequiredArgsConstructor
+public enum ServerErrors implements ServerError {
+  TOKEN_UNAUTHORIZED("token.unauthorized"),
+  UNKOWN("unknown");
```
Comment (KevinFHartmann) [inline song-client/src/main/java/org/icgc/dcc/song/client/errors/ServerErrors.java]: Unknown

File: song-client/src/main/java/org/icgc/dcc/song/client/register/RestClient.java [hunk 2]
```diff
@@ -65,21 +73,26 @@ public Status post(String url, String json) {
...
+      }
+    } catch (Exception e){
+      status.err(e.toString());
     }
+
     return status;
   }
 
   public Status put(String url, String json) {
     Status status = new Status();
```
Comment (KevinFHartmann) [inline song-client/src/main/java/org/icgc/dcc/song/client/register/RestClient.java]: val

File: song-server/src/test/java/org/icgc/dcc/song/server/service/existence/ExistenceServiceTest.java [hunk 3]
```diff
@@ -30,7 +30,7 @@
 @SpringBootTest
 @RunWith(SpringRunner.class)
 @ContextConfiguration(classes = RetryConfig.class)
-@ActiveProfiles({ "dev", "secure" })
+@ActiveProfiles("dev")
 public class ExistenceServiceTest {
 
   private static final int PORT = 8080;
```
Comment (KevinFHartmann) [inline song-server/src/test/java/org/icgc/dcc/song/server/service/existence/ExistenceServiceTest.java]: PORT = 9876

File: song-client/src/main/java/org/icgc/dcc/song/client/cli/Status.java [hunk 4]
```diff
@@ -66,11 +71,25 @@ public void output(String format, Object... args) {
...
+      AnsiConsole.out().println(
+          ansi()
+              .eraseLine()
+              .fg(GREEN)
+              .a(outputs)
+              .reset());
+    }
   }
 
   public void report() {
```
Comment (KevinFHartmann) [inline song-client/src/main/java/org/icgc/dcc/song/client/cli/Status.java]: 👍

File: song-server/src/test/java/org/icgc/dcc/song/server/service/existence/ExistenceServiceTest.java (hunk 3 above)
Comment (rtisma) [inline song-server/src/test/java/org/icgc/dcc/song/server/service/existence/ExistenceServiceTest.java]: ahahha yes enough with the conflicting 8080

**Comment by rtisma** [comment]
not done yet, this is just so stuff if building

**Comment by rtisma** [comment]
@KevinFHartmann im going to delete this PR since it was intended for client-error handling but ended up being server error handling. as a result, i have cloned this branch into a new one called server-error handling, and will be deleting this one

**Pull request closed without merging**